common/lib/chem/chem/tests.py

```diff
@@ -223,247 +220,250 @@ def test_divide_fract_to_frac_wrog(self):
 
 
 class Test_Render_Equations(unittest.TestCase):
-
+    """
+    Tests to validate the HTML rendering of plaintext (input) equations
+    """
+    # pylint: disable=line-too-long
     def test_render1(self):
-        s = "H2O + CO2"
-        out = render_to_html(s)
+        test_s = "H2O + CO2"
```

Comment:
This addresses the letter of the law, but not the spirit.

What does `s` stand for? `test_string`?